schemas/didDocument/didDocument.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$id": "/did-core.didDocument",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "title": "@context",
+        "description": "https://w3c.github.io/did-core-registry/#context",
+        "anyOf": [
+          { "type": "string" },
+          { "type": "array",
+            "items": [
+              {
+                "type": "string",
+                "const": "https://www.w3.org/ns/did/v1"
+              }
```

Comment:
Don't remember if this is correct JSON Schema for 1..n, where the first item MUST equal "https://www.w3.org/ns/did/v1"... I think what this is saying is "only one item where the first item is "https://www.w3.org/ns/did/v1""... and don't have time to look it up.

Reply:
You can confirm it here: https://www.jsonschemavalidator.net/

I would prefer to have actual tests enforce this kind of thing.

I can add tests both positive and negative for JSON Schema properties, if thats acceptable... I recommend we start now rather than try and do this later.

Reply:
The incredibly out of date test suite is here: https://github.com/w3c-ccg/did-test-suite

Reply:
I mean adding a github action to this repo that uses the json schema definitions and ensures that changes to them are not breaking... so that in addition to reviewing a PR for text errors, a machine can ensure that a PR has not broken JSON Schema definitions... or JSON-LD... if that is stored here.... 

The test suite you are referencing does not protect the did core registry, I'm suggesting we create a did core registry test suite that protects the machine readable definitions from human error. 

We could also add interoperability tests so we could easily see the delta between methods, but I'm not sure that is appropriate for this registry, although its absence might encourage method implementers to ignore this repo, and not contribute to interoperability definitions.

I'm personally in favor of having 2 test suites in this repo:

1. did core registry test suite (protects json schema, json-ld, cbor defintions hosted here).
2. did interoperability test suite (allows method implementers to prove compliance with the registry, similar to the test suite we have at the DIF https://identity.foundation/context/test-report.html)

Reply:
> You can confirm it here: https://www.jsonschemavalidator.net/

Confirmed... although I was able to get some garbage through the schema... [see fix below](https://github.com/w3c/did-core-registry/pull/4/files#r384774457).

Reply:
> I mean adding a github action to this repo that uses the json schema definitions and ensures that changes to them are not breaking

Yes, +1 -- what you're really talking about is just doing validation on the JSON Schema files?

Hmm, you might also be talking about doing validation using the JSON Schema files on a sample set of DID Documents  and DIDs from a variety of DID Methods? +1 to automating that too.

So, basically, you have to pass both test suites in order to be able to click the rebase button (or the tests need to be changed).

Reply:
yes, I don't want to add that here, but I can open a separate PR to do so.